media/css/firefox/android/all.less

```diff
@@ -0,0 +1,100 @@
+@import '../../sandstone/lib.less';
+
+#main-feature {
+    .span-all();
+    h1 {
+        margin-bottom: @baseLine;
+    }
+    h2 {
+        .font-size(24px);
+        letter-spacing: -0.25px;
+        line-height: 1.4;
+    }
+}
+
+#main-content {
+    .build-table-container {
+        .span-all();
+    }
+    table {
+        width: 100%;
+        thead {
+            th {
+                white-space: nowrap;
+                &:not([colspan]) {
+                    width: 15%;
+                }
+            }
+            th, td {
+                padding: 10px 0;
+            }
+        }
+        tbody {
+            th, td {
+                padding: 10px 0;
+            }
+        }
+        .download {
+            padding: 10px 0;
+            a {
+                display: block;
+                padding: 5px 5px 5px 35px;
+                background-position: left center;
+                background-repeat: no-repeat;
+                background-image: url(/media/img/firefox/android/android-download.svg);
```

Comment:
You could here do:

```
background: transparent url(/media/img/firefox/android/android-download.svg) left center no-repeat;
```
Are we concerned with IE8 support on this page? The reason I ask is because IE8 does not support SVG :-/